app/views/my_projects/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= t('my_projects.header') %></h1>
+
+<h2><%= t('my_projects.approved_projects_header') %></h2>
```

Comment:
Hunh! I'd been visualizing this distinction as some kind of label within the project summary module, rather than grouping under headers, but while I think I still like my assumption better I don't actually have objections to this (especially as a first pass.) Thoughts on which one you like better?

Reply:
I don't have a preference, but will probably stay with the headings for now.